WangscapeTest/TestMetaOutput.cpp

```diff
@@ -12,17 +12,18 @@
 #include <tilegen/TilesetGenerator.h>
 #include <tilegen/partition/TilePartitionerSquares.h>
 
-class TestMetaOutput : public ::testing::Test {
+#include "TestRequiringOptions.h"
+
+class TestMetaOutput : public TestRequiringOptions
+{
 protected:
-    std::string filename;
+    using TestRequiringOptions::TestRequiringOptions;
```

Comment:
Yes, that's probably better. Although I would prefer to keep the check that prevents `filename` from being set more than once. GTest tests are after all supposed to be independent.